File: toDoApp.py
# toDoApp.py
# Applied Black formatter 

tasks = []


def addtask(task):
    tasks.append(task)
    print("task added!")


def showTasks():
    if len(tasks) == 0:
        print("no tasks yet")
    else:
        for i in range(len(tasks)):
            print(i + 1, ".", tasks[i])


def removetask(tasknumber):
    tasks.pop(tasknumber - 1)
    print("task removed!!")


def main():
    while True:
        print("1. Add Task")
        print("2. Show Tasks")
        print("3. Remove Task")
        print("4. Exit")
        ch = input("enter choice : ")
        if ch == "1":
            t = input("enter task : ")
            addtask(t)
        elif ch == "2":
            showTasks()
        elif ch == "3":
            showTasks()  
            if tasks:  
                try:
                    n = int(input("enter task no to remove: "))
                    if 1 <= n <= len(tasks):
                        removetask(n)
                    else:
                        print("Invalid task number!")
                except ValueError:
                    print("Please enter a valid number!")
            else:
                print("No tasks to remove!")
        elif ch == "4":
            break
        else:
            print("wrong choice!!")
        print() 


main()

File: ToDoApp.java
// ToDoApp.java
import java.util.*; 

public class ToDoApp {
    static ArrayList tasks = new ArrayList();

    public static void addtask(String t){
        tasks.add(t);
        System.out.println("Task Added!!"); 
    }

    public static void showTasks( ) {
        if(tasks.size()==0){
            System.out.println("No tasks yet"); 
        }
        else {
            for(int i=0;i<=tasks.size();i++){ 
                System.out.println((i+1)+". "+tasks.get(i));
            }
        }
    }

    public static void removeTask(int n) {
        tasks.remove(n); 
        System.out.println("task removed"); 
    }

    public static void main(String args[]) {
        Scanner s=new Scanner(System.in);
        while(true){
            System.out.println("1 Add Task"); // <-- inconsistent spacing
            System.out.println("2.Show Tasks");
            System.out.println("3.Remove Task");
            System.out.println("4 Exit");
            int choice=s.nextInt(); // 
            if(choice==1){
                System.out.println("Enter Task: "); 
                String t=s.nextLine(); 
                addtask(t);
            } else if(choice==2){
                showTasks();
            } else if(choice==3){
                System.out.print("Enter task no to remove: ");
                int n=s.nextInt(); 
                removeTask(n); // <--
            } else if(choice==4){
                break; 
            } else{
                System.out.println("Wrong Choice!!");
            }
        }
    }
}
